src/include/platform/CHIPDeviceConfig.h

```diff
@@ -532,6 +532,26 @@
 #define CHIP_DEVICE_CONFIG_ENABLE_JUST_IN_TIME_PROVISIONING 0
 #endif
 
+// -------------------- Service Discovery Configuration -----------------------
+
+/**
+ * CHIP_DEVICE_CONFIG_ENABLE_MDNS
+ *
+ * Enable support to use MDNS for service advertising and discovery in CHIP.
+ */
+#ifndef CHIP_DEVICE_CONFIG_ENABLE_MDNS
```

Comment:
After second thought, maybe it would be better if we set GN's chip_enable_mdns based on Kconfig, and not this define. The reason is that `chip_enable_mdns` not only affects `CHIP_DEVICE_CONFIG_ENABLE_MDNS`, but can also be used to control which files are built. What do you think?

Reply:
Or we can always fixed when it's needed... Nevermind then.